### PR TITLE
changing ngen to produce be both x64 and x86 images instead of the vs…

### PR DESCRIPTION
**Pull request by jmarolf**
… default (just x86)

@Huy-Nguyen-MSFT @Pilchie @ManishJayaswal @mavasani @agocke @dotnet/roslyn-compiler @dotnet/roslyn-infrastructure

### CODE REVIEW

File: src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr [hunk 1]
```diff
@@ -8,10 +8,10 @@ vs.dependencies
   vs.dependency id=Microsoft.Net.4.6.1.FullRedist.Resources
 
 folder InstallDir:\MSBuild\15.0\Bin
-    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngen=yes
-    file source=$(OutputPath)\csc.exe vs.file.ngen=yes
-    file source=$(OutputPath)\csi.exe vs.file.ngen=yes
-    file source=$(OutputPath)\vbc.exe vs.file.ngen=yes
+    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngenArchitecture=all vs.file.ngenPriority=1
```
Comment (Huy-Nguyen-MSFT) [inline src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr]: Should the priority be 3 instead? 1 means the NGEN operations will be done immediately, which will increase installation time. vs.file.ngen=yes has the default value of 3.

Comment (Pilchie) [inline src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr]: Agree that it should probably be pri 3.

**Comment by Pilchie** [comment]
Scenario approved - please ensure two code reviews and successful testing.

**Comment by jmarolf** [comment]
@et13 can you also take a look

**Comment by brettfo** [comment]
If you remove the priority it'll default to 3.

:shipit:

**Comment by Pilchie** [comment]
@Huy-Nguyen-MSFT - can you take another look?

**Comment by ManishJayaswal** [comment]
This needs more code reviews as per @Pilchie and I need more details from @jmarolf on what testing has been done before I approve it.

**Comment by jmarolf** [comment]
test  windows_release_unit32_prtest please

File: src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr [hunk 2]
```diff
@@ -8,10 +8,10 @@ vs.dependencies
   vs.dependency id=Microsoft.Net.4.6.1.FullRedist.Resources
 
 folder InstallDir:\MSBuild\15.0\Bin
-    file source=$(OutputPath)\VBCSCompiler.exe vs.file.ngen=yes
```
Comment (jaredpar) [inline src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr]: What is the point of `ngen=yes` if it's really `ngen=maybe`?  

Have we given feedback to the developers here on the time spent tracking down this problem?

Comment (jmarolf) [inline src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr]: `ngen=yes` means *use the default ngen setting for vs*.  Apparently they didn't what to have all that text in the option.

Comment (jmarolf) [inline src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr]: I actually think `ngen=default` would be a better name.  Then you would at least need to lookup what the default ngen values are.

Comment (TyOverby) [inline src/Setup/DevDivVsix/CompilersPackage/Microsoft.CodeAnalysis.Compilers.swr]: I wonder if we could do a search on other .swr files in the VS repo and see if we can't find other people making the same mistake?

File: src/Tools/MicroBuild/publish-assets.ps1 [hunk 3]
```diff
@@ -41,6 +41,7 @@ try
     switch ($branchName)
     {
         "dev15-rc" { } 
+        "refs/pull/14800/head" {}
```
Comment (jmarolf) [inline src/Tools/MicroBuild/publish-assets.ps1]: removing...

**Pull request merged**